### PR TITLE
Fix chart sync

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
This change will ensure that we'll always sync the charts to the kubermatic-installer repo with the latest tag for each branch.

Additionally it will remove the `version_values.yaml` as it caused confusion now in several situations.

This must also be backported to v2.7 & v2.8

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -138,6 +128,8 @@ pipeline:
   sync-charts:
     commands:
     - apk add --no-cache -U git bash openssh
+    - export LATEST_VERSION=$(git describe --tags | cut -d '-' -f 1)
+    - sed -i "s/{KUBERMATIC_TAG}/${LATEST_VERSION}/g" config/kubermatic/values.yaml
```
Comment (alvaroaleman) [inline .drone.yml]: What is `$KUBERMATIC_TAG` here?

Comment (alvaroaleman) [reply .drone.yml]: Ah got it. Can you please use `__KUBERMATIC_TAG__` instead of `{KUBERMATIC_TAG}` ? Ppl like me will read `${KUBERMATIC_TAG}` here

Comment (mrIncompetent) [reply .drone.yml]: Done with https://github.com/kubermatic/kubermatic/pull/2137/commits/a039a52c0afeac7edf5b6c76fa9c3fbb9d170228

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**